doc/source/notebooks/bullet_world.ipynb

```diff
@@ -1 +0,0 @@
-../../../examples/bullet_world.ipynb
```

Comment:
Why did you delete this ? Could you please put it back, you can do this with the terminal command "ln -s <path-to-pycram>/examples/bullet_world.ipynb <path-to-pycram>/doc/source/notebooks/bullet_world.ipynb"